tests/setup.js

```diff
@@ -8,6 +8,9 @@ Object.keys(document.defaultView).forEach(function mapProperties(property) {
     global[property] = document.defaultView[property];
   }
 });
+console.error = (message) => {
```

Comment:
this only works on React 16 for this specific bug, but it catches others

Reply:
It looks like you are failing lint? Also if this starts getting too rabbit hole-y, maybe push the console.error stuff into a seperate PR so this bug fix can go out?

Reply:
yea agreed, I'll remove this and ship it